package.json

```diff
@@ -115,7 +116,7 @@
     "@nrwl/linter": "^10.0.7",
     "@nrwl/node": "^10.0.7",
     "@nrwl/nx-plugin": "^10.0.7",
-    "@nrwl/workspace": "^10.0.7",
+    "@nrwl/workspace": "^10.2.1",
```

Comment:
please revert the changes in the package-lock.json and in the `package.json`. You have to have all `@nrwl/` dependencies on the same version.

Reply:
I would bump the version of nrwl depenendcies in a seperate PR

Reply:
Thank you for the thorough review, I think now there are no changes to the current package.json in the pr.